### PR TITLE
Updated

### PR DESCRIPTION
**Pull request by akcays**
closes #41

### CODE REVIEW

**Comment by akcays** [comment]
closes #41

File: recipely-server/config/config.js [hunk 1]
```diff
@@ -0,0 +1 @@
+module.exports = '6a637c79023fc91e5eebbd087157b560';
```
Comment (gzeegz) [inline recipely-server/config/config.js:1]: The .gitignore file isn't ignoring this.

**Review by gzeegz — COMMENTED** [review]
Where is the updated package.json that has "axios" as a dependency?

File: recipely-server/config/config.js (hunk 1 above)
Comment (gzeegz) [reply recipely-server/config/config.js:1]: Also, create a "config.example.js" that shows how the config file should look. The "config.example.js" will be committed to GitHub, and not "config.js".

**Pull request closed without merging**